monkey/monkey_island/cc/ui/src/components/ui-components/Select.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
How many times the options are being rendered?

Reply:
Many times

Reply:
I think parent components rerender causing it to rerender too